docs/packaging/updating-an-existing-package.md

```diff
@@ -167,6 +167,20 @@ Full release notes:
 - [ ] Package was built and tested against unstable
 ```
 
+### Other commit message format examples
+
+In the cases where you are not updating a package to a new version, but simply applying a patch of fixing an existing version, the following commit message shortlogs are considered suitable:
+
+- Add the `homepage` metatdata key:
+  `package: Add homepage`
+- Fix a packaging issue (including adding a patch to fix the package)
+  `package: Fix (...)`
+- Updating a README, monitoring.yml file or other stuff that does not directly _change_ the package, meaning the package release number didn't need a bump:
+  `package: [NFC] Add README.md and monitoring.yml`
```

Comment:
I don't think we have documentation for README.md and monitoring.yml yet. Would it more prudent to use MAINTAINERS.md as an example?